analysis/org.eclipse.tracecompass.analysis.profiling.core/src/org/eclipse/tracecompass/analysis/profiling/core/callstack/CallStackAnalysis.java

```diff
@@ -54,6 +54,13 @@ public abstract class CallStackAnalysis extends TmfStateSystemAnalysisModule imp
      */
     public static final String CALL_STACK = "CallStack"; //$NON-NLS-1$
 
+    /**
+     * Annotations string state attribute
+     *
+     * @since 2.6
+     */
+    public static final String ANNOTATIONS = "Markers"; //$NON-NLS-1$
```

Comment:
This shouldn't be here because it's state provider parameter and CallStackAnalysis is replaced by InstrumentedCallstackAnalysis. It be part of the CallStackStateProvider class instead.